t/t3404-rebase-interactive.sh

```diff
@@ -955,6 +955,17 @@ test_expect_success 'rebase -ix with --autosquash' '
 	test_cmp expected actual
 '
 
+test_expect_success 'rebase -i with --allow-inline-reword' '
```

Comment:
Since this verifies that `--allow-inline-reword`, this test should probably be squashed into the second commit that adds support for that option.